pandas/core/series.py

```diff
@@ -1446,6 +1446,50 @@ def to_string(self, buf=None, na_rep='NaN', float_format=None, header=True,
     def iteritems(self):
         """
         Lazily iterate over (index, value) tuples.
+
+        This method returns a zip of tuples (index, value). This is useful When
+        one want to create new series from the values of an old one. Be aware
```

Comment:
"one want**s**" (but "you want" probably sounds better)

Reply:
I'm not sure about creating a new series using itertuples. I'd rather say something about lazily iterating over the index, value pairs.

Reply:
I am not sure to perfectly get what you mean. I tried to add an example where vectorization is not possible (as far as I know maybe I am wrong)